sno/diff-view.html

```diff
@@ -125,7 +125,7 @@
             }).addTo(map)
 
             var layerGroup = L.featureGroup()
-            for (let [dataset, diff] of Object.entries(DATA['sno.diff/v1'])) {
+            for (let [dataset, diff] of Object.entries(DATA['sno.diff/v1+hexwkb'])) {
```

Comment:
how's it going to view wkb in the browser? Surely needs the GeoJSON output?

Reply:
I did this by accident, so good catch. I'm surprised this is using the _json_ diff and not the _geojson_ diff. That throws a spanner in the works a bit, because I can't just switch tot he geojson one, it's quite different. Will attempt to though.

Reply:
It's using the JSON because the geojson ends up in multiple files

Reply:
Alternatively, it could serialise all the geojson into an array in the HTML or something. No problem doing that, but it needs to work :)

Reply:
The `--geojson` diff doesn't actually differentiate multiple datasets, so it's not enough for this.

So I guess to make this work, I need to add a format variant for `--json` diffs with GeoJSON geometries?

Reply:
Don’t understand... the geojson gets written to multiple files in a directory?

Reply:
aha, so it does (if you have >1 dataset). I just saw it with one dataset and got lost on how the different ones were going to be differentiated, but didn't realise it used multiple _files_

Reply:
otherwise there's no meaningful way to view them in QGIS/etc

Reply:
fixed in cb28c3b - it now writes geojson files to a temp dir and then reads them back in again. _Could_ have done it some other way, e.g. by passing an empty dict to `resolve_object_path` and making it create a StringIO for each dataset, but it felt like a Weird Hack so I didn't